testcontainers-demo/testcont_test.go

```diff
@@ -0,0 +1,41 @@
+//go:build testcont
+// +build testcont
+
+package main
+
+import (
+	"context"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/testcontainers/testcontainers-go/modules/mongodb"
+	"github.com/testcontainers/testcontainers-go/modules/redis"
+)
+
+func TestServerWithTestcontainers(t *testing.T) {
+	ctx := context.Background()
+
+	mongodbContainer, err := mongodb.Run(ctx, "docker.io/mongodb/mongodb-community-server:7.0-ubi8")
```

Comment:
is there any considerations in term of things installed locally for this framework to run a container and download images ? 
Like do you need the VM / pod / machine  to have docker install or anything else ?

Reply:
or is it completely self managed via that golang pkg ?

Reply:
https://java.testcontainers.org/supported_docker_environment/

To run Testcontainers-based tests, you need a Docker-API compatible container runtime, such as using [Testcontainers Cloud](https://www.testcontainers.cloud/) or installing Docker locally.